routes/index.js

```diff
@@ -53,25 +53,25 @@ exports.admin = function (req, res, next) {
 };
 
 function parse(todo) {
-  var t = todo;
+  var t1 = todo;
```

Comment:
Prefer using `const` and `let` over `var` to define variables to avoid scope-related issues.